src/every_query/eval.py

```
@@ -0,0 +1,100 @@
+    task_set_dir = Path(cfg.task_set_dir)
+    if not task_set_dir.is_dir():
+        raise NotADirectoryError(f"{task_set_dir} is not a directory")
+
+    codes = list(map(str, cfg.codes))
+    if not codes:
+        raise ValueError("cfg.codes is empty")
+
+    rows: list[dict[str, Any]] = []
+
+    for code in codes:
+        slug = code_slug(code)
```

Ruff (0.14.8)

Undefined name `code_slug`

(F821)